java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -690,6 +690,200 @@ void testInnerJoinWithOnlyCommonKeys() {
     }
   }
 
+  @Test
+  void testLowerBound() {
+    ColumnVector columnVector = null;
+    Table table = null;
+    Table values = null;
+    boolean[] descFlags = new boolean[3];
+    try {
```

Comment:
nit: I think these tests could be a lot smaller if we used `assertColumnsAreEqual`

```
try (Table table = new Table.TestBuilder()
                               .column(10, 20, 20, 20, 20)
                               .build();
       Table values = new Table.TestBuilder().column(20).build();
       ColumnVector expected = ColumnVector.fromBoxedInts(0);
       ColumnVector lower = table.lowerBound(values, descFlags, true)) {
  assertColumnsAreEqual(expected, lower);
}
```